src/components/UI/Button.tsx

```diff
@@ -0,0 +1,32 @@
+import { classnames } from '@/utils'
+import { FC, ReactNode } from 'react'
+
+type ButtonProps = {
+  children: ReactNode
+  className?: string
+  disabled?: boolean
+  type?: 'button' | 'reset' | 'submit'
+}
+
+const THEMES = {
```

Comment:
Should a common type be here too?

Reply:
You mean type for THEMES?

Reply:
yes

Reply:
Will do this when implementing theme functionality